stable/atlantis/README.md

```diff
@@ -0,0 +1,67 @@
+| `bitbucket.base_url`                        | Bitbucket Server only: The hostname of your Bitbucket Server installation.                                                                                                                                                                                                                                                                                 | n/a                    |
+| `gitconfig`                                 | Contents of a file to be mounted to `~atlantis/.gitconfig`.  Use to allow redirection for Terraform modules in private git repositories.                                                                                                                                                                                                                   | n/a                    |
+| `aws.credentials`                           | Contents of a file to be mounted to `~atlantis/.aws/credentials`.                                                                                                                                                                                                                                                                                          | n/a                    |
+| `aws.config`                                | Contents of a file to be mounted to `~atlantis/.aws/config`.                                                                                                                                                                                                                                                                                               | n/a                    |
+| `serviceAccountSecrets.credentials`         | JSON object representing secrets for a Google Cloud Platform production service account.  Only applicable if hosting Atlantis on GKE.                                                                                                                                                                                                                      | n/a                    |
+| `serviceAccountSecrets.credentials-staging` | JSON object representing secrets for a Google Cloud Platform staging,service account.,Only applicable if hosting Atlantis on GKE.                                                                                                                                                                                                                          | n/a                    |
+
+## Testing the Deployment
+
+To perform a smoke test of the deployment (i.e. ensure that the Atlantis UI is up and running):
+
+1.  Install the chart.  Supply your own values file or use `test-values.yaml`, which has a minimal set of values required in order for Atlantis to start.
+
+    ```bash
+    helm install -f test-values.yaml --name my-atlantis . --debug
```

Comment:
helm install requires a name of the chart too

Reply:
Fixed.